mk/rabbitmq-run.mk

```diff
@@ -65,7 +65,7 @@ node_feature_flags_file = $(call node_tmpdir,$(1))/feature_flags
 node_enabled_plugins_file = $(call node_tmpdir,$(1))/enabled_plugins
 
 # Broker startup variables for the test environment.
-HOSTNAME := $(shell hostname -s)
+HOSTNAME := $(shell U="$$(uname)"; if [ "$$U" = Darwin -o "$$U" = FreeBSD  ]; then hostname -s; else hostname; fi)
```

Comment:
Erlang.mk already provides `$(PLATFORM)`. You can use GNU Make directives instead of a shell script, this is most of the time more efficient (even though we end up running a shell command in this case).

So something like:
```
ifeq ($(PLATFORM),msys2)
HOSTNAME := $(shell hostname)
else
HOSTNAME := $(shell hostname -s)
endif
```

Speaking about shell scripts, I'm wondering if we can use `$(COMPUTERNAME)` on Windows instead of running `hostname`. What do you think?

Reply:
Great ideas, let me modify this PR and test them out.